react-native/react/menubar/index.js

```diff
@@ -1,20 +1,43 @@
 /* @flow */
+/*eslint-disable react/prop-types */ // Since we're using flow types for props
 
 import React, {Component} from '../base-react'
 import Render from './index.render'
 import {connect} from '../base-redux'
 
+import {remote, shell} from 'electron'
+import {ipcRenderer} from 'electron'
+
 export type MenubarProps = {
-  debug?: boolean
+  username: ?string,
+  debug: ?boolean
 }
 
 class Menubar extends Component {
   props: MenubarProps;
   render () {
-    return <Render {...this.props}/>
+    const openingMessage = this.props.username ? 'Be sure to drink your Ovaltine.' : "Looks like you aren't logged in. Try running `keybase login`"
+
+    const closeMenubar = () => ipcRenderer.send('closeMenubar')
+
+    // TODO change this to /keybase when all our mount points are `/keybase/...`
+    // TODO Support linux
+    const kbfsPath = `${process.env.HOME}/keybase`
```

Comment:
so this moved to /keybase already, but ultimately we should get this from the daemon. its fine for this to be hardcoded now but likely we should move it out